### PR TITLE
feat: support gitpod

### PR DESCRIPTION
**Pull request by qsliu2017**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
feature

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**
developers can start coding at `https://gitpod.io/#/github.com/4paradigm/OpenMLDB` (my forked repo has setup [gitpod](https://gitpod.io/#/github.com/qsliu2017/OpenMLDB))
this reduces dependency install and first-time building

need helps:
- [ ] install Gitpod GitHub app on this repo, see https://www.gitpod.io/docs/prebuilds#github

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+tasks:
+  - before: |
+      sudo apt update -y
+      DEBIAN_FRONTEND=noninteractive sudo apt-get install -y bison python3-dev libcppunit-dev build-essential cmake autoconf tcl pkg-config git curl patch libtool-bin
+
+      # install bazel
+      sudo apt install -y apt-transport-https curl gnupg
+      curl -fsSL https://bazel.build/bazel-release.pub.gpg | gpg --dearmor > bazel.gpg
```
Comment (aceforeverd) [inline .gitpod.yml]: we use bazelisk for zetasql.
have a look in https://github.com/4paradigm/zetasql/blob/feat/hybridse-zetasql/.github/workflows/main.yml#L87

File: .gitpod.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+tasks:
+  - before: |
+      sudo apt update -y
+      DEBIAN_FRONTEND=noninteractive sudo apt-get install -y bison python3-dev libcppunit-dev build-essential cmake autoconf tcl pkg-config git curl patch libtool-bin
```
Comment (aceforeverd) [inline .gitpod.yml]: make sure python3 is the default python interpreter

Comment (qsliu2017) [reply .gitpod.yml]: sure, the default image [gitpod/workspace-full](https://hub.docker.com/r/gitpod/workspace-full) sets python3 as default

File: .gitpod.yml (hunk 1 above)
Comment (qsliu2017) [reply .gitpod.yml]: this follows the [Download Pre-Compiled Thirdparty](https://openmldb.ai/docs/en/main/deploy/compile.html#:~:text=Method%20Two%3A%20Download%20Pre%2DCompiled%20Thirdparty). is bazel/bazelisk necessary in this method?

Comment (aceforeverd) [reply .gitpod.yml]: not necessary, bazel only required if you want to compile thirdparty (zetasql) from source

File: .gitpod.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
```
Comment (aceforeverd) [inline .gitpod.yml]: `bison, libcppunit-dev, tcl, pkg-config, patch, libtool-bin` also not required if just download pre-compiled thirdparty

**Review by aceforeverd — APPROVED** [review]

**Comment by aceforeverd** [comment]
@lumianph see if we're ok to install gitpod ? gitpod is similar to GitHub Code Space but is in public service so probably more developer can use it than code space

**Review by lumianph — APPROVED** [review]

**Comment by gitpod-for-oss** [comment]
> @lumianph see if we're ok to install gitpod? gitpod is similar to GitHub Code Space but is in public service so probably more developer can use it than code space

Installing [Gitpod GitHub App](https://github.com/apps/gitpod-io) to this repository will help you to [enable prebuilds](https://www.gitpod.io/docs/prebuilds/) to keep always ready to code.

Also, In Gitpod you can use Multiple IDEs & not just VS Code. [Detailed difference](https://www.gitpod.io/vs/github-codespaces)

**Pull request merged**